src/log4net/ObjectRenderer/RendererMap.cs

```diff
@@ -217,7 +211,7 @@ public IObjectRenderer Get(Type type)
   /// <summary>
   /// Gets the default renderer instance
   /// </summary>
-  public IObjectRenderer DefaultRenderer => _sDefaultRenderer;
+  public static IObjectRenderer DefaultRenderer { get; } = new DefaultRenderer();
```

Comment:
For record: this change in public API broke log4net 3.0.3 for log4net.Ext.Json 3.0.1.

Reply:
reason for #211